pkg/render/compliance.go

```diff
@@ -30,6 +30,9 @@ import (
 	v3 "github.com/tigera/api/pkg/apis/projectcalico/v3"
 	operatorv1 "github.com/tigera/operator/api/v1"
 	"github.com/tigera/operator/pkg/components"
+	rdata "github.com/tigera/operator/pkg/render/common/data"
+	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
```

Comment:
WDYT about `res` for this? `relasticsearch` feels a bit long and it reads like a typo to me but that could just be me, personally I'd rather it be an acronym than something that reads "odd".
I think this is just preference so no change necessary.

Reply:
Why not just "elasticsearch", or if we really feel we need to alias it do "es"? I'm not sure we need the alias, though.

I don't like "res" because that's a common abbreviation for "resource".

Reply:
It doesn't feel too long to me, but I think I'm more OK with longer names than most developers. @caseydavenport I'd like to keep the prefix so that it is clear that this is for rendering Elasticsearch components and not working with Elasticsearch, even if there isn't a package collision. It just keeps things consistent among all packages.